src/main/java/org/kainos/ea/api/DeliveryEmployeeService.java

```diff
@@ -0,0 +1,6 @@
+package org.kainos.ea.api;
+
+public class DeliveryEmployeeService extends EmployeeService{
```

Comment:
Empty class?

Reply:
Just a based to be used by other team memebers